tools/c7n_gcp/tests/test_spanner.py

```diff
@@ -357,3 +357,21 @@ def test_set_iam_policy_remove_bindings_star(self):
                               'members': ['user:dkhanas@gmail.com']}]
 
         self.assertEqual(test_method(existing_bindings, bindings_to_remove), expected_bindings)
+
+
+class TestIamSpannerInstance(BaseTest):
+
+    def test_query(self):
+        project_id = 'cloud-custodian'
+        session_factory = self.replay_flight_data('test-iam-spanner-instance',
+                                                  project_id=project_id)
+        policy = self.load_policy(
+            {'name': 'iam-spanner-instance',
+             'resource': 'gcp.spanner-instance'},
```

Comment:
this isn't testing query against a backup, but a spanner instance

Reply:
right, sorry. It's fixed